pages/common/pg_ctl.md

```diff
@@ -3,13 +3,13 @@
 > Utility for controlling a PostgreSQL server and database cluster.
 > More information: <https://www.postgresql.org/docs/current/app-pg-ctl.html>.
 
-- Start a PostgreSQL server:
+- Initialize a new PostgreSQL database cluster:
 
-`pg_ctl -D {{data_directory}} -l {{log_file_name}}`
```

Comment:
Why remove the log file directive, though?

Reply:
It is generally a better practice to configure logging directly in the postgresql.conf rather than on the command line.

Reply:
Thanks for clarifying. :)